Selective merge of feature-egm branch into master

rtkg opened the pull request:
*Changes*

* no dependencies on the unreleased abb_egm_interface and abb_rws_interface packages in yumi_hw and yumi_cameras  
* added vel/pos controllers for both arms in yumi_control/config
* added traj controllers for both arms in yumi_control/config
* included install directives in the CMakeList.txt files which are necessary for a future release
* removed KTH accessories and KTH setup related features in yumi_description and yumi_launch

*TODO*

Tested only in simulation - test on the actual robot with the launch files in yumi_launch; might need changes in yumi_support/yumi_hw

Commit:
selective merge of feature-egm branch into master (w/o unreleased ABB code dependencies)

YoshuaNava commented:
Hey,
I'll be running tests on the real robot tomorrow. I'll keep you updated on the results.

YoshuaNava committed:
Some fixes to have abb_driver working with the grippers

Ports for abb_driver: 11000-11002, 12000-12002
Ports for grippers: 13000-13002

Grippers node launches with the robot interface now

MoveIt! wait for the left/right arm controllers directly

YoshuaNava commented:
I just checked this pull request, and fixed some network parameters, to have yumi_support running with interfaces for left arm, right arm, and the grippers.

I also commented the ROS controllers action servers from MoveIt! controllers.yaml, so that its ready to use faster.

For running yumi_support we need industrial core MAX_NUM_JOINTS to be equal to 10. We should add a comment regarding this somewhere.


About the RWS interface to ROS control (not yumi_support), we haven't used it at KTH. Do you also want me to check it? Are we providing support for it?

YoshuaNava committed:
Some changes to make position control through RWS work

YoshuaNava committed:
YuMi firmware for RWS interface

YoshuaNava committed:
Updating launch files
